test/zdtm/static/seccomp_filter_tsync.c

```diff
@@ -158,6 +160,7 @@ int main(int argc, char **argv)
 		 * inherited.
 		 */
 		if (filter_syscall(__NR_ptrace, SECCOMP_FILTER_FLAG_TSYNC) < 0)
+			pthread_mutex_unlock(&getpid_wait);
```

Comment:
Have you tested these changes? _exit is called unconditionally...

Reply:
I don't know why but I am trying my best to resolve these